README.md

```diff
@@ -46,7 +46,7 @@ to your code, and then `go [build|run|test]` will automatically fetch the necess
 Otherwise, run the following Go command to install the `gin` package:
 
 ```sh
-$ go get -u github.com/gin-gonic/gin
+$ go install github.com/gin-gonic/gin@latest
```

Comment:
`go install` will not update the `go.mod` file, suggest keeping using `go get -u`.